_pages/setup-wallets/paper-advanced.md

```diff
@@ -60,12 +60,7 @@ Expert users only: <https://github.com/mflaxman/human-rng-golang>
 #### Improve Airgap
```

Comment:
Would you like me to remove this entire `Improve Airgap` section as it isn't really required anymore if you use a QR code instead of downloading the JSON file.

Reply:
I think it makes sense eventually, but for now QR isn't fully converged on. Hopefully soon!